pact/ledger.pact

```diff
@@ -110,6 +110,12 @@
     (compose-capability (UPDATE_SUPPLY))
   )
 
+  (defconst MARMALADE_GUARD "marmalade-guard")
```

Comment:
This doesn't need to be its own def as that value is never used outside of `ledger-guard`

Reply:
Once https://github.com/kadena-io/pact/issues/807 is fixed we can create module guard consts